### ISSUE TITLE
How to configure

### ISSUE THREAD

**Issue by 51qzpw** [PBSPro/kubernetes-pbspro-connector#3]
Where should I fill in the sample configuration?
![image](https://user-images.githubusercontent.com/44865193/77849508-ad384780-71fe-11ea-972d-cff79ab3a664.png)

**Comment by saimanikant** [comment]
Hi @51qzpw ,
Are you referring to the Hook's config? If yes, you need to add the path of the argument --config passed to kubelet.

**Comment by 51qzpw** [comment]
Yes,I want to use this hook to my kubernetes cluster,However, I did not successfully configure according to readmin.md, I had a single-node kubernetes cluster, and PBS was installed on the node. When compiling the scheduler, I modified the apiHost in kubernetes.go to be my local kube_apiserver 192.168.1.55:6443, and then what do I need to do?

**Comment by saimanikant** [comment]
Ok, so before enabling the the hook, please modify the hook's config file and set value to the parameter kubelet_config.
Then run the custom scheduler as root, for a pod to get scheduled.

**Comment by scottaltair** [comment]
Change directory to kubernetes-pbspro-connector folder. 
```bash
cd kubernetes-pbspro-connector
```

Update `pbs_kubernetes.CF` with the absolute path to the kubelet config directory. 
The value to --config is the absolute path to the directory that the kubelet will watch for pod manifests to run. This directory will need to be created before starting the scheduler.
```bash
{
    "kubelet_config": "/aboslute/path/to/kubelete_config"
}
```

Install PBS Pro hook and config file
```bash
qmgr << EOF
create hook pbs-kubernetes
set hook pbs-kubernetes event = execjob_end
set hook pbs-kubernetes event += execjob_launch
import hook pbs-kubernetes application/x-python base64 pbs_kubernetes.PY
import hook pbs-kubernetes application/x-config base64 pbs_kubernetes.CF
EOF
```
